nRF Toolbox/Connection/ConnectionViewController.swift

```diff
@@ -32,7 +32,7 @@
 
 import UIKit
 
-protocol ConnectionViewControllerDelegate: class {
+protocol ConnectionViewControllerDelegate: AnyObject {
```

Comment:
why AnyObject?

Reply:
It's the future!
https://forums.swift.org/t/class-only-protocols-class-vs-anyobject/11507/5